### ISSUE TITLE
Unconditionally connect to database from DATABASE_URL env when opting for Docker

### ISSUE THREAD

**Issue by webyneter** [pydanny/cookiecutter-django#1541]
E.g. in `config.settings.base`:

```python
DATABASES = {
    'default': env.db('DATABASE_URL'),
}
```

**Comment by wandering-tales** [comment]
I currently use Docker for local development, but removing the default value Travic CI builds fail.

**Comment by browniebroke** [comment]
Yes, the Travis config isn't suitable for Docker currently. See #2100

**Comment by wandering-tales** [comment]
Great! Good to know the issue has been reported.

I tried to apply the changes proposed in #1950. But unfortunately that was not enough. My current workaround was to provide the following default value for the `DATABASE_URL` environment variable, not explicitly set in non-Docker deployments:

```python
DATABASES = {
    'default': env.db('DATABASE_URL', default='postgres:///{{cookiecutter.project_slug}}'),
}
```

I wonder why that default has been removed in 6c26d39f65a66429ae1ec9a00d9676f29c408c45.
